composer/utils/iter_helpers.py

```diff
@@ -88,3 +90,37 @@ def iterate_with_pbar(iterator, progress_bar=None):
             yield x
             if pb is not None:
                 pb.update(len(x))
+
+
+def iter_to_stream(iterator: Iterator[bytes], buffer_size: int = io.DEFAULT_BUFFER_SIZE) -> io.BufferedReader:
```

Comment:
Could we remove the function, and instead just have the class? See comment below.